tests/test_server.py

```diff
@@ -104,13 +105,16 @@ def test_version(rasa_app: SanicTestClient):
 
 def test_status(rasa_app: SanicTestClient):
     _, response = rasa_app.get("/status")
+    model_file = response.json["model_file"]
     assert response.status == 200
     assert "fingerprint" in response.json
     assert "model_file" in response.json
+    assert "models/" in model_file or os.path.isfile(response.json["model_file"])
```

Comment:
so this doesn't work?
```suggestion
    os.path.isfile(response.json["model_file"])
```

Reply:
I'll check it again I don't think it did I need to remember why, I'll check it when I get home later from traveling.  I also thought we wanted to also verify the proper model dir part as well in the response or no?

Reply:
It does work with just the isfile check but shouldn't we also verify the path we get back is also the `models/model.tar.gz` and not the tmp location?